angular-ngrx-scss/src/app/shared/components/repo-controls/repo-controls.component.html

```diff
@@ -52,7 +53,7 @@
           {{ (selectSortFilter$ | async)?.split('_')?.join(' ') | lowercase }}
         </span>
       </p>
-      <span class="results__clear-container">
+      <span class="results__clear-container" (click)="handleClearClick()">
         <div class="close-background">
```

Comment:
Same as below - a div inside inline element is not semantic. Also I would rename the classes to respect BEM convention:

- `results__clear-container` -> `results__clear`
- `results__clear` -> can be left empty or something like `results__clear__text` 
- `close-background` -> `results__clear__icon`

Reply:
changed